scene/gui/tab_bar.cpp

```diff
@@ -149,6 +160,7 @@ void TabBar::gui_input(const Ref<InputEvent> &p_event) {
 					offset--;
 					_update_cache();
 					update();
+					update_minimum_size();
```

Comment:
Is `update_minimum_size()` necessary here? `buttons_visible` is only true with clipped tabs and minimum size doesn't change then.

Reply:
Yes. The tab's elements (style and icons) influence the minimum height, and if a tall tab not shown before is made visible, the minimum size should be updated.